setup.py

```diff
@@ -77,6 +77,7 @@
 install_requires = [
     'Flask-BabelEx>=0.9.2',
     'Flask>=0.11.1',
+    'celery<4.0,>=3.1',
```

Comment:
If I am not mistaken there is no way to install it with out celery, https://github.com/inveniosoftware/invenio-webhooks/blob/3c7e987f0a49544a7c3a31460e02d36bd965368f/invenio_webhooks/models.py#L32